src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
 
-    { I18N_TNMT_SELECT_TEAM1, "Choose with up and down, select with fire." },
-    { I18N_TNMT_SELECT_TEAM2, "Select Team" },
-    { I18N_TNMT_JOIN_RED_TEAM, "${1} joins the red team" },
-    { I18N_TNMT_JOIN_BLUE_TEAM, "${1} joins the blue team" },
+    { I18N_TNMT_SELECT_TEAM1, "Choisissez avec 'haut' et 'bas', sélectionnez avec 'tirer'." },
+    { I18N_TNMT_SELECT_TEAM2, "Sélectionner une équipe" },
+    { I18N_TNMT_JOIN_RED_TEAM, "${1} à rejoint l'équipe rouge" },
+    { I18N_TNMT_JOIN_BLUE_TEAM, "${1} à rejoint l'équipe bleue" },
 
-    { I18N_TNMT_TEAM_RED_CAP, "TEAM RED" },
-    { I18N_TNMT_TEAM_BLUE_CAP, "TEAM BLUE" },
+    { I18N_TNMT_TEAM_RED_CAP, "ÉQUIPE ROUGE" },
+    { I18N_TNMT_TEAM_BLUE_CAP, "ÉQUIPE BLEUE" },
 
-    { I18N_SERVE_WARM_UP, "warm up" },
-    { I18N_SERVE_GAME_BEGINS, "game begins" },
-    { I18N_SERVE_CONNECT, "${1} connected" },
-    { I18N_SERVE_RECONNECT, "${1} reconnected" },
-    { I18N_SERVE_SPECTATING, "${1} is spectating now" },
-    { I18N_SERVE_UNKNOWN_COMMAND, "Unknown server command: ${1}" },
-    { I18N_SERVE_WRONG_PASSWORD, "Wrong admin password" },
-    { I18N_SERVE_ALREADY_ADMIN, "You're already admin" },
-    { I18N_SERVE_NEW_ADMIN, "${1} is now server admin" },
-    { I18N_SERVE_ADMIN_LEFT, "${1} left the server admin mode" },
-    { I18N_SERVE_KICK_YOURSELF, "You cannot kick yourself" },
-    { I18N_SERVE_PLAYER_NOT_FOUND, "Player not found: ${1}" },
-    { I18N_SERVE_NOT_AUTHORIZED, "You're not authorized to do that" },
-    { I18N_SERVE_KICK, "${1} kicked ${2}" },
-    { I18N_SERVE_NOT_IMPLEMENTED, "Not implemented yet" },
-    { I18N_SERVE_NEXT_MAP, "${1} started next map" },
-    { I18N_SERVE_MAP_USAGE, "Usage: /map <map_name> <warmup> <duration>" },
-    { I18N_SERVE_MAP_LOADED, "${1} started map ${2}" },
-    { I18N_SERVE_CONFIG_RELOADED, "configuration reloaded" },
-    { I18N_SERVE_CONFIG_SAVED, "configuration saved" },
-    { I18N_SERVE_GET_USAGE, "Usage: /get <variable>" },
-    { I18N_SERVE_SET_USAGE, "Usage: /set <variable> <value>" },
-    { I18N_SERVE_SET_VAR_TO, "set ${1} to ${2}" },
-    { I18N_SERVE_RESET_USAGE, "Usage: /reset <variable>" },
-    { I18N_SERVE_VAR_CLEARED, "${1} cleared" },
-    { I18N_SERVE_PARM_MISSING, "Parameter missing" },
-    { I18N_SERVE_NO_PARM_NEEDED, "No parameters needed" },
-    { I18N_SERVE_ILLEGAL_PARM, "Illegal parameters" },
+    { I18N_SERVE_WARM_UP, "échauffement" },
+    { I18N_SERVE_GAME_BEGINS, "la partie commence" },
+    { I18N_SERVE_CONNECT, "${1} s'est connecté" },
+    { I18N_SERVE_RECONNECT, "${1} s'est re-connecté" },
+    { I18N_SERVE_SPECTATING, "${1} est maintenant spectateur" },
+    { I18N_SERVE_UNKNOWN_COMMAND, "Commande server inconnue: ${1}" },
+    { I18N_SERVE_WRONG_PASSWORD, "Mot de passe administrateur erroné" },
+    { I18N_SERVE_ALREADY_ADMIN, "Vous êtes déja administrateur" },
+    { I18N_SERVE_NEW_ADMIN, "${1} est maintenant administrateur" },
+    { I18N_SERVE_ADMIN_LEFT, "l'administrateur ${1} a quitté le serveur" },
```

Comment:
Actually the original message means that the admin left the "admin mode", e.g. like de-OP'ing yourself on IRC :) So maybe something like "${1} n'est plus dans le mode d'administration du serveur".